src/Microsoft.DotNet.Wpf/tests/UnitTests/PresentationCore.Tests/System/Windows/ClipboardTests.cs

```diff
@@ -103,8 +114,8 @@ public void SetData_EmptyOrWhitespaceFormat_ThrowsArgumentException(string? form
 
     [WpfFact]
     public void SetData_Null_Throws()
-    {
-        Action action = () => Clipboard.SetData("MyData", data: null!);
+        {
```

Comment:
The error format?

Reply:
Yes, I'll catch this with the next update I make.